examples/tutorials/05_creating_CNN.ipynb

```diff
@@ -43,9 +43,7 @@
     "from batchflow.models.tf import TFModel\n",
```

Comment:
<p>Replace <code>examples.utils.utils</code> to <code>batchflow.utils</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>